setup.py

```diff
@@ -419,7 +419,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
 
 _devel_only_amazon = [
     "aws_xray_sdk",
-    "moto[cloudformation,glue]>=4.2.5",
+    "moto[cloudformation,glue]>4.2.5",
```

Comment:
```suggestion
    "moto[cloudformation,glue]>4.2.9",
```

See https://github.com/getmoto/moto/issues/7031 which I raised them yesterday and they fixed it few hours ago. It's not `strictly` needed, it would have fixed itself in CI after successful main build. But it's good to add it for local development - however you really need to add 4.2.9 because only this version **really** fixes the problem